packages/protvista-utils/src/TrackHighlighter.js

```diff
@@ -8,6 +8,7 @@ const combineRegions = (region1, region2) => {
 export default class TrackHighlighter {
   constructor({ element, min, max }) {
     this.element = element;
+    this.element._highlightcolor = "#FFEB3BCC";
```

Comment:
This could be set as a `const` at the top of the file:

`const DEFAULT_HIGLIGHT_COLOR="#FFEB3BCC"`